src/qml/util.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <qml/util.h>
+
+#include <string>
+
+#include <QQuickWindow>
+#include <QSGRendererInterface>
+#include <QString>
+
+namespace QmlUtil {
+
+QString GraphicsApi(QQuickWindow* window)
+{
+    switch (window->rendererInterface()->graphicsApi()) {
+    case QSGRendererInterface::Unknown: return "Unknown";
+    case QSGRendererInterface::Software: return "The Qt Quick 2D Renderer";
+    case QSGRendererInterface::OpenGL: return "OpenGL ES 2.0 or higher";
+    case QSGRendererInterface::Direct3D12: return "Direct3D 12";
+    case QSGRendererInterface::OpenVG: return "OpenVG via EGL";
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
```

Comment:
There is the [isApiRhiBased function](https://doc-snapshots.qt.io/qt6-dev/qsgrendererinterface.html#isApiRhiBased). It was introduced in Qt 5.14. 

This check is then appropriate and allows for a simplified control flow.

Reply:
@jarolrod still needs `#if` as the cases below don't exist in <5.14.

Reply:
yep, i meant that the way it is done here simplifies the control flow compared to if we used the function.